geoips/commandline/geoips_test.py

```diff
@@ -140,7 +141,7 @@ def add_arguments(self):
             "--outdir",
             "-o",
             type=str,
-            default=f"{environ['GEOIPS_OUTDIRS']}",
+            default=f"{PATHS['GEOIPS_OUTDIRS']}",
```

Comment:
Does this need to be an f-string? Couldn't it just be the value?

Reply:
Changed.